config/initializers/global.rb

```diff
@@ -10,5 +10,6 @@
 #
 TRANSACTION_TYPE=[
   "Withdraw",
```

Comment:
@babarajay , Why do you think it should be in the initializer? Is there any other better place?

Reply:
I'll remove it